static/webpack-config.js

```diff
@@ -25,11 +26,31 @@ module.exports = function () {
     });
   }
 
+  const globalConfigPath = `./${jamboConfig.dirs.config}/global_config.json`;
+  let globalConfig = null;
```

Comment:
if we default globalConfig to empty object we can change `useJwt = globalConfig && globalConfig.useJWT` to just `useJWT = globalConfig.useJWT` or even `{ useJWT } = globalConfig`